### PR TITLE
replace telefonica peru with movistar peru as isp name

### PR DESCRIPTION
**Pull request by AndraMeer**

### CODE REVIEW

File: content/geoip/release-notes/2024.mdx [hunk 1]
```diff
@@ -77,7 +77,7 @@ and branding conventions:
 | `Versatel Deutschland`              | `1&1 Versatel`                   |
 | `Telefonica O2 UK`                  | `O2 UK`                          |
 | `INDOSAT Internet Network Provider` | `Indosat`                        |
-| `Telefonica del Peru`               | `Telefonica Peru`                |
+| `Telefonica del Peru`               | `Movistar Peru`                  |
```
Comment (dhogan8) [inline content/geoip/release-notes/2024.mdx:80]: Nitpick, but there's some additional white space after `Movistar Peru`

Comment (AndraMeer) [reply content/geoip/release-notes/2024.mdx:80]: so i added it to make the vertical lines at the end line up, they didn't until I did that -- I checked and they all have extra spacing added :)

Comment (AndraMeer) [reply content/geoip/release-notes/2024.mdx:80]: i've just had another change come through for this same release note so i'm going to add it to this PR if that's okay, we can review again tomorrow.

Comment (AndraMeer) [reply content/geoip/release-notes/2024.mdx:80]: @dhogan8 this is ready for another look when you have a bit of time :) Thanks!

Comment (dhogan8) [reply content/geoip/release-notes/2024.mdx:80]: Oops! My bad and good catch.

**Review by dhogan8 — APPROVED** [review]

File: content/geoip/release-notes/2024.mdx (hunk 1 above)
Comment (AndraMeer) [reply content/geoip/release-notes/2024.mdx:80]: not at all, i appreciate it!! thanks!!

**Pull request merged**